resources/core/requirements.yaml

```diff
@@ -30,4 +30,6 @@ dependencies:
   - name: docs
```

Comment:
nit: uncomment the components?

Reply:
Def. Yes! It was just a temp. action to be able to upgrade the Core `helm` chart in a running local kyma installation and it was complaining of them 👍